public/content/mu-plugins/mu-imagify-helper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Plugin Name: Imagify upload path controller.
+ * Description: Provide the correct upload root paths in a multisite environment using Dekodes custom path definitions.
+ * Version:     0.1.0
+ * Author:      Dekode
+ *
+ * Inspired by the `Imagify Custom Site Root` plugin.
+ */
+
+declare( strict_types = 1 );
+
+namespace Dekode\MUPlugins;
+
+/**
+ * Conditionally override the Imagify plugin root path.
+ *
+ * Our custom setup has the upload directory outside the WordPress root, and also names it differently from `wp-content`.
+ * This confuses the Imagify plugin when in a multisite environment, working on a primary site, so some magic is needed here.
+ *
+ * @param string $root_path
+ * @return string;
+ */
+function imagify_site_root_override( $root_path ) {
+	$upload_basedir = imagify_get_filesystem()->get_upload_basedir( true );
```

Comment:
The imagify plugin might change in the future. Should we add a check to see if this function exists before using it?